### DISCUSSION TITLE
Onyx Boox handwriting optimisation for Saber

### DISCUSSION THREAD

**Discussion by ZweckMa** [saber-notes/saber#1152] [Q&A]
It would be amazing to be able to use Saber on Onyx Boox Tablets. 
I managed to get handwriting support for Squid, wich is not supported by Onyx themselfs.
https://gist.github.com/calliecameron/b3c62c601d255630468bd493380e3b7e
Has someone some idea what a "drawViewKey" with Saber might be?

**Comment by jdkruzr** [comment]
just want to re-up this, I think there either is or was previously a branch of the software with Onyx SDK support built in. this would be a massive boost for Boox users.
